lib/actuators/src/DCMotor.cpp

```diff
@@ -0,0 +1,25 @@
+#include "DCMotor.h"
+
+#include <cmath>
+
+DCMotor::DCMotor::DCMotor(PinName pwm, PinName dir, bool inverted, int freqInHz, float limit)
```

Comment:
Can someone explain why we are using namespaces to wrap the classes? If it's just to include the 'config' with each class couldn't we just have a namespace for the different configs? Code like the stuff above seems strange to me with 3x "DCMotor"

Reply:
@younesr1

Reply:
its a cpp best practice. google recommends it and a lot of other resources do too. it prevents polluting global namespace which is important. and yeah it looks strange at first but well get used to it